documentation/modules/adding-virt-provider.adoc

```diff
@@ -10,14 +10,16 @@ You can add {a-virt} destination provider to the {ocp} web console in addition t
 
 .Prerequisites
 
-* You must have {a-virt} link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/authentication_and_authorization/using-service-accounts[service account token] with `cluster-admin` privileges.
+* You must have {a-virt} link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/authentication_and_authorization/using-service-accounts[service account token] with sufficient permissions on {project-short} resources such as providers and plans and on Openshift Virtualization resources such as `VirtualMachine` and `DataVolume`.
```

Comment:
For remote migration, there is no need to have permissions on MTV resources, generally, there is no MTV tool installed on the remote host.

```suggestion
* You must have {a-virt} link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/authentication_and_authorization/using-service-accounts[service account token] with sufficient permissions on Openshift Virtualization resources such as `VirtualMachine` and `DataVolume`.
```

Reply:
@qiyuann yes, that's true - migrations and plans were not good examples as they are created on the local OCP cluster, but we cannot limit this only to Openshift Virtualization resources because in case of imports from vSphere we also create the conversion pod there and we need to be able to create a secret - is there anything else actually?

Reply:
actually, maybe leaving it as it was ("service account token with `cluster-admin` privileges") is ok for now - we didn't change it in 2.4 but we didn't find the exact set of permissions required so we can change it when we have that set. @qiyuann WDYT?

Reply:
It would requires permissions on `configmaps, events, namespaces, persistentvolumeclaims, persistvolumes, pods, serets, jobs, network-attachment-definitions, storageclasses, virtualmachines, datavolumes, securitycontextconstraints `

But I agree with leaving it as it was, which makes it much easier for users to apply.

Reply:
@ahadas @qiyuann So it should be "You must have {a-virt} link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/authentication_and_authorization/using-service-accounts[service account token] with `cluster-admin` privileges." Correct?

Reply:
@ahadas @qiyuann I reverted the changes to the prerequisite. Please take another look.